sdk_container/src/third_party/coreos-overlay/sys-libs/libselinux/libselinux-3.5.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="7"
-PYTHON_COMPAT=( python3_{6..10} )
-USE_RUBY="ruby25 ruby26 ruby27"
+PYTHON_COMPAT=( python3_{10..11} )
```

Comment:
I think we have already updated python eclasses to include python 3.12 as a known implementation, so there should be no need for dropping python 3.12 from PYTHON_COMPAT (one Flatcar modification less).